collector/perf_linux.go

```diff
@@ -14,17 +14,24 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
-	"runtime"
-
 	perf "github.com/hodgesds/perf-utils"
 	"github.com/prometheus/client_golang/prometheus"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+	"runtime"
+	"strconv"
+	"strings"
```

Comment:
Normally we group all the built-in imports at the top, can you re-format these using `goimports`?
```
go get golang.org/x/tools/cmd/goimports
goimports -w ./collector/perf_linux.go
```